pages/api/v1/auth/actions/changeUsername.ts

```diff
@@ -13,17 +13,16 @@ import messages from '~src/auth/utils/messages';
 
 async function handler(req: NextApiRequest, res: NextApiResponse<ChangeResponseType | MessageType>) {
 	if (req.method !== 'POST') return res.status(405).json({ message: 'Invalid request method, POST required.' });
-
-	const body = JSON.parse(req.body);
-	const { username, password } = body;
-	if(!body || !username || !password) return res.status(400).json({ message: 'Missing parameters in request body' });
+	const body = JSON.parse(JSON.stringify(req.body));
```

Comment:
we don't need to parse the body here with `NextApiRequest` type. It is already parsed